looprpc/client.proto

```diff
@@ -1645,28 +1661,53 @@ enum DepositState {
     */
     WITHDRAWN = 3;
 
+    /*
+    LOOPING_IN indicates that the deposit is currently being used in a static
+    address loop-in swap.
+    */
+    LOOPING_IN = 4;
+
+    /*
+    LOOPED_IN indicates that the deposit was used in a static address loop-in
+    swap.
+    */
+    LOOPED_IN = 5;
+
+    /*
+    SWEEP_HTLC_TIMEOUT indicates that the deposit is part of an active loop-in
+    of which the respective htlc was published by the server and the timeout
+    path has opened up for the client to sweep.
+    */
+    SWEEP_HTLC_TIMEOUT = 6;
+
+    /*
+    HTLC_TIMEOUT_SWEPT indicates that the timeout path of the htlc has been
+    swept by the client.
+    */
+    HTLC_TIMEOUT_SWEPT = 7;
+
     /*
     PUBLISH_EXPIRED indicates that the deposit has expired and the sweep
     transaction has been published.
     */
-    PUBLISH_EXPIRED = 4;
+    PUBLISH_EXPIRED = 8;
```

Comment:
Like reasoned here https://github.com/lightninglabs/loop/pull/786#discussion_r1716810761 I thought it might help us with debugging issues as we know exactly which route a deposit took through the state machine (without looking at the state update tables).